src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -962,13 +1012,9 @@ def _test_replica_mgmt(self, resource_group, skuName, testSkuName, isBasicTier):
                           '--admin-user cloudsa --admin-password SecretPassword123 '
                           '--sku-name {}'
                           .format(database_engine, resource_group, server, skuName),
-                          checks=[
```

Comment:
May i know the reason decreasing properties to check?

Reply:
Previously, we used to output all server properties to the user many of which were irrelevant. We have now reduced that to limited information which the user would find useful. replication capacity is one such field which added very little value to the user. So, we removed that from the create output. And since validating that parameter does not really help much, we have removed from validating that field from the replication tests as well.